### PR TITLE
Add Solution provider docs

### PR DESCRIPTION
**Pull request by maayarosama**
### Description

Adding solution provider docs

### Related Issues
#124

### CODE REVIEW

File: wiki/solution_provider/solution_provider_chain.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# How to use the solution provider with your deployments?
+
+After fllowing the step for creating your solution provider as mentioned [here](solution_provider.md).
```
Comment (rawdaGastan) [inline wiki/solution_provider/solution_provider_chain.md]: please change fllowing to following

File: wiki/solution_provider/solution_provider_chain.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+You can now add the solutionProviderId to your deployment as follows:
+
+- ### Grid3_client
+```bash
+vm.solutionProviderID= solutionProviderID;
```
Comment (rawdaGastan) [inline wiki/solution_provider/solution_provider_chain.md]: have a space before =

File: wiki/solution_provider/solution_provider_chain.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# How to use the solution provider with your deployments?
+
+After fllowing the step for creating your solution provider as mentioned [here](solution_provider.md).
+
+You can now add the solutionProviderId to your deployment as follows:
+
+- ### Grid3_client
```
Comment (rawdaGastan) [inline wiki/solution_provider/solution_provider_chain.md:7]: why making ### Grid3_client a point

File: wiki/solution_provider/solution_provider_chain.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+```bash
+vm.solutionProviderID= solutionProviderID;
+```
+This should be added your deployments as shown bellow
```
Comment (rawdaGastan) [inline wiki/solution_provider/solution_provider_chain.md]: This should be added `to` your deployments as shown `below`

File: wiki/solution_provider/solution_provider_chain.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+![add_solutionProviderID](./img/grid3_solution.png)
+
+After deployment the machine, The solutionProviderID will be shown in the machine specs
```
Comment (rawdaGastan) [inline wiki/solution_provider/solution_provider_chain.md]: After `deploying` the machine

**Review by rawdaGastan — CHANGES_REQUESTED** [review]

File: wiki/solution_provider/solution_provider_chain.md (hunk 3 above)
Comment (maayarosama) [reply wiki/solution_provider/solution_provider_chain.md:7]: Because we'll add another example using terraform

**Review by rawdaGastan — APPROVED** [review]

**Pull request merged**